update-postgres.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# Script to update the database
+# The script creates a file db_backup.bck that stores the generated dump
+# The script is designed to have generous sleep times to not break on slower servers
+
+# NB: IF YOU TRY STARTING THE WRONG MAJOR VERSION CONTAINER AT FIRST THE SCRIPT WILL CURRENTLY FAIL LEADING TO DATA LOSS
+
+if [ -f /.dockerenv ]; then
+  echo "This script cannot be run inside a docker container.";
+  exit 1;
+fi
+
+version=$1
+config_file=${2:-$USER.env}
+
+if [ -z "$version" ]; then
+  echo "Please provide the desired PostgreSQL version as the first argument"
+  exit 1
+fi
+
+# Set config file
+if [ ! -f "$config_file" ]; then
+  echo "Config file $config_file not found"
+  exit 1
+fi
+
+# Check if the required environment variables are set
+if [ -z "$DATE_POSTGRESQL_VERSION" ] || [ -z "$DATE_DB_PORT" ] || [ -z "$DATE_DB_PASSWORD" ]; then
+  echo "Error: Required environment variables are not set"
+  exit 1
+fi
+
+# Make sure website is stopped
+docker compose down && sleep 15 && docker-compose up db -d
```

Comment:
Our server still runs an old version of Docker and "docker compose" will need to be switched to "docker-compose"

Reply:
Should be fixed in latest commit